src/bootstrap/Cargo.toml

```diff
@@ -52,6 +52,7 @@ walkdir = "2"
 
 # Dependencies needed by the build-metrics feature
 sysinfo = { version = "0.26.0", optional = true }
+suggest-tests = { version = "0.1.0", path = "../tools/suggest-tests" }
```

Comment:
This isn't what I meant - it still rebuilds bootstrap when you modify the code. I was imagining that you would compile it separately and then parse the paths it outputs in bootstrap. Right now your `bootstrap_tool!` line in `tool.rs` is never used.
```
(bash@dev-desktop-us-1.infra.rust-lang.org) ~/rust3 [18:10:09]
> touch src/tools/suggest-tests/src/static_suggestions.rs 
(bash@dev-desktop-us-1.infra.rust-lang.org) ~/rust3 [18:10:16]
> x suggest --run
Building rustbuild
   Compiling suggest-tests v0.1.0 (/home/gh-jyn514/rust3/src/tools/suggest-tests)
   Compiling bootstrap v0.0.0 (/home/gh-jyn514/rust3/src/bootstrap)
    Finished dev [unoptimized] target(s) in 3.90s
```